tasks/LaundrySort.tex

```diff
@@ -0,0 +1,44 @@
+\section{Laundry Sort [Housekeeper]}
+The robot is helping to do laundry by sorting it.
+
+% \subsection{Focus}
+% This test focuses on object perception, manipulation, and planning.
+
+\subsection{Main Goal}
+The robot is given a bin of laundry and must sort it by color and size. There are both children's and adult's clothing in the bin, and the robot must make piles by color and size. There are existing piles of laundry on the floor into which the laundry in the bin must be sorted.
```

Comment:
Isn't it usually the other way around? I pick up laundry once a day from piles in various rooms which are usually: next to my bed, in the master bathroom, and next to the baby's changing table into 3 bins- one for each temperature of laundry. 40, 60 and 95 (degrees C).

Reply:
I may be doing laundry wrong.

I put similarly-colored items together, and then I wash them all on cold in the gentle cycle.

The only exceptions are truly filthy things like gym socks or camping gear.

Reply:
I’ve seen both systems and many others that would raise an eyebrow.

The correct way according to manufacturers is separate by textile type (e.g. cotton, polyester, wool, etc) and by colour. Most people separates only by owner and washes the underare apart.

Whatever it is, this is overly specified. Let’s start with clolour only

Reply:
I’ve seen both systems and many others that would raise an eyebrow.

The correct way according to manufacturers is separate by textile type (e.g. cotton, polyester, wool, etc) and by colour. Most people separates only by owner and washes the underare apart.

Whatever it is, this is overly specified. Let’s start with clolour only